config.md

```diff
@@ -49,6 +49,14 @@ For example, if a configuration is compliant with version 1.1 of this specificat
 }
 ```
 
+### Example (Windows)
+
+```json
+"root": {
+    "path": "\\\\?\\Volume{ec84d99e-3f02-11e7-ac6c-00155d7682cf}"
```

Comment:
> The trailing backslash doesn't actually matter.

So say that in the docs?  Perhaps:

> … MUST be specified as a [volume GUID path][naming-a-volume] (a trailing backslash is OPTIONAL)…

[naming-a-volume]: https://aka.ms/nb3hqb